packages/dapp-svelte-wallet/react-ui/src/App.js

```diff
@@ -1,22 +1,85 @@
 /* eslint-disable react/display-name */
 import React from 'react';
+import { Switch, Route } from 'react-router-dom';
+import { CssBaseline } from '@material-ui/core';
+import {
+  createTheme,
+  makeStyles,
+  ThemeProvider,
+} from '@material-ui/core/styles';
 
-import WalletConnection from './components/WalletConnection';
+import AppBar from './components/AppBar';
+import NavMenu from './components/NavMenu';
+import Contacts from './views/Contacts.js';
+import Dapps from './views/Dapps.js';
+import Dashboard from './views/Dashboard.js';
+import Purses from './views/Purses.js';
+import Issuers from './views/Issuers.js';
 
-import './App.css';
-import { withApplicationContext } from './contexts/Application';
+const appBarHeight = '64px';
+const navMenuWidth = '240px';
 
-const App = ({ connectionState }) => {
+const appTheme = createTheme({
+  palette: {
+    background: {
+      default: '#ffffff',
+    },
+  },
+  typography: {
+    fontFamily: ['"Roboto"', '"Helvetica"', '"Arial"', 'sans-serif'].join(','),
+    fontWeightRegular: 500,
+    h1: {
+      fontFamily: ['"Montserrat"', '"Arial"', 'sans-serif'].join(','),
+      fontSize: '32px',
+      fontWeight: '700',
+      letterSpacing: '-1.5px',
+      lineHeight: '48px',
+      margin: 0,
+    },
+  },
+  appBarHeight,
+  navMenuWidth,
+});
+
+const useStyles = makeStyles(_ => ({
+  main: {
+    boxSizing: 'border-box',
+    padding: '32px',
+    marginLeft: navMenuWidth,
+    position: 'absolute',
+    width: `calc(100vw - ${navMenuWidth})`,
+    top: appBarHeight,
+  },
+}));
+
+const App = () => {
+  const classes = useStyles();
   return (
-    <div className="App">
-      <header className="App-header">
-        Connection Status: {connectionState}
-      </header>
-      <WalletConnection></WalletConnection>
-    </div>
+    <ThemeProvider theme={appTheme}>
+      <CssBaseline />
+      <NavMenu />
+      <main className={classes.main}>
+        <Switch>
+          <Route path="/purses">
```

Comment:
Are these Route paths compatible with installing the wallet under `http://localhost:8000/wallet/`?

Reply:
No I don't believe so. We can specify the basename "/wallet" as documented here https://reactrouter.com/web/api/BrowserRouter/basename-string to make that work. However, I'll hold off on that until we make the react router the default. We probably want to do it dynamically depending on how the wallet is being loaded (local dev vs agoric open).